tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -54,30 +101,26 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 			Expect(err).ToNot(HaveOccurred(), "Error removing label from operator "+info.OperatorPrefix)
 		}
 		installedLabeledOperators = nil
-
-		By("Delete csvs marked for deletion")
-		for _, csv := range csvsToDelete {
-			err := affiliatedcerthelper.DeleteCsv(csv.OperatorPrefix, csv.Namespace)
-			Expect(err).ToNot(HaveOccurred(), "Error deleting csv "+csv.OperatorPrefix)
-		}
-		csvsToDelete = nil
-
 	})
 
 	// 46699
-	It("one operator to test, operator does not belong to certified-operators organization in Red Hat catalog [skip]",
+	It("one operator to test, operator does not belong to certified-operators organization in Red Hat catalog [negative]",
 		func() {
-			// operator is already installed.
-			// not deleting csv yet because it is also used in the next test case
-
 			By("Label operator to be certified")
 
 			err := affiliatedcerthelper.AddLabelToInstalledCSV(
-				affiliatedcertparameters.UncertifiedOperatorPrefixNginx,
-				affiliatedcertparameters.ExistingOperatorNamespace,
+				affiliatedcertparameters.UncertifiedOperatorPrefixFalcon,
+				affiliatedcertparameters.TestCertificationNameSpace,
 				affiliatedcertparameters.OperatorLabel)
 			Expect(err).ToNot(HaveOccurred(), "Error labeling operator "+
-				affiliatedcertparameters.UncertifiedOperatorPrefixNginx)
+				affiliatedcertparameters.UncertifiedOperatorPrefixFalcon)
+
+			// add falcon operator info to array for cleanup in AfterEach
+			installedLabeledOperators = append(installedLabeledOperators, affiliatedcertparameters.OperatorLabelInfo{
```

Comment:
Is it possible to avoid using installedLabeledOperators between test cases and AfterEach section. It would be perfect if we could make test cases more isolated.